src/helpers/JWT.js

```diff
@@ -3,15 +3,21 @@ import { SECRET_KEY, EXPIRATION_DURATION } from '../config/constants';
 /**
  * Class handles JWT authentication
  */
-class JWT {
+export default {
   /**
-   * Method to generate token from userId and role
+   * Method to generate token
    * @param {object} data
    * @returns {string} generated token
    */
-  static async generateToken(data) {
-    const token = await jwt.sign({ key: data }, SECRET_KEY, { expiresIn: EXPIRATION_DURATION });
+  generateToken: (data) => {
+    const token = jwt.sign({ key: data }, SECRET_KEY, { expiresIn: EXPIRATION_DURATION });
     return token;
-  }
-}
-export default JWT;
+  },
```

Comment:
Just a suggestion, since we agreed to use functional style. Exporting object literals, is that not just another syntax for the object class style.

Reply:
It's still functional since the function is being exported. This is just another way to export if one doesn't want to do `export default function(){}`